kubernetes/config-maps/concent-api/local_settings.py.j2

```diff
@@ -30,6 +30,7 @@ ADMINS = [
 EMAIL_HOST          = "{{ email.host }}"
 EMAIL_PORT          = {{ email.port }}
 EMAIL_HOST_USER     = "{{ email.user }}"
+EMAIL_SUBJECT_PREFIX = "[concent-api-{{ email.subject }}] "
```

Comment:
Let's do it a bit differently, without adding a new variable. Please remove the `email.subject`. Then change the value to: `"[{{ gke.cluster }}:concent-api] "`.